screen-attributions/src/main/java/com/ivy/attributions/AttributionsViewModel.kt

```diff
@@ -0,0 +1,118 @@
+package com.ivy.attributions
+
+import androidx.compose.runtime.Composable
+import com.ivy.core.ComposeViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class AttributionsViewModel @Inject constructor() :
+    ComposeViewModel<AttributionsState, AttributionsEvent>() {
+    private val attributionItems = listOf<AttributionItem>(
```

Comment:
1) Move this to a separate class let's say `AttributionsProvider`. This class will have `provideAtrributions` function.

2) You need to use `ImmutableList` and use `persistentListOf()`